packages/core/test/unit/reporters/mutation-test-report-helper.spec.ts

```diff
@@ -540,14 +540,34 @@ describe(MutationTestReportHelper.name, () => {
           status: MutantStatus.NoCoverage,
           fileName: 'add.js',
           id: '3',
-          location: factory.location(),
+          location: { start: { column: 1, line: 1 }, end: { column: 1, line: 1 } },
           mutatorName: 'fooMutator',
           replacement: '"bar"',
           coveredBy: ['1'],
           static: false,
         };
         expect(actual).deep.include(expected);
       });
+
+      it('should not modify the original location property', () => {
+        // Arrange
+        const input = factory.mutantTestCoverage({
+          fileName: 'add.js',
+          id: '3',
+          location: factory.location(),
```

Comment:
You probably don't want to rely on the defaults for this test, since you assert them below. I always try to be explicit in the tests.